await-generator/src/SOFe/AwaitGenerator/RaceLostException.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * await-generator
+ *
+ * Copyright (C) 2018 SOFe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+declare(strict_types=1);
+
+namespace SOFe\AwaitGenerator;
+
+use Exception;
+
+/**
+ * The default exception to throw into an async iterator
+ * when `Traverser::interrupt()` is called.
```

Comment:
```suggestion
 * The exception to throw into loser generators of
 * a {@link Await::safeRace()}.
 * 
 * If your generator has side effects, please consider
 * handling it so cancellation can be taken. NOTICE: 
 * it would not cause a crash even though your
 * generator did not catch it.
```